grpc/src/main/java/io/stargate/grpc/service/interceptors/NewConnectionInterceptor.java

```diff
@@ -83,13 +83,14 @@ public NewConnectionInterceptor(
               .headers(stringHeaders)
               .remoteAddress(call.getAttributes().get(Grpc.TRANSPORT_ATTR_REMOTE_ADDR))
               .build();
+      Context context = Context.current().withValue(GrpcService.HEADERS_KEY, stringHeaders);
 
-      Connection connection = connectionCache.get(info);
+      if (!"stargate.Stargate/getSchemaChanges"
+          .equals(call.getMethodDescriptor().getFullMethodName())) {
+        Connection connection = connectionCache.get(info);
+        context = context.withValue(GrpcService.CONNECTION_KEY, connection);
+      }
```

Comment:
This is not super elegant...
Maybe creating the connection doesn't matter that much, since connections are cached and the same client is likely to invoke other methods than this one.